tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -281,6 +281,25 @@ Verify RHODS Display Name and Version
     Should Be Equal       ${rhods_version_t[1]}   ${rhods_version}   msg=RHODS vesrion and label is not consistent
     Should Be Equal       ${rhods_displayname}   Red Hat OpenShift Data Science  msg=Dieplay name doesn't match
 
+Verify RHODS Notebooks Network Policies
+    [Documentation]    Verifies that the network policies for RHODS Notebooks are present on the cluster
+    [Tags]    Smoke    Tier1
```

Comment:
either smoke or tier1, it shouldn't be both

Reply:
done